rapid7/metasploit-framework
# Allow msftidy to run as a post-merge hook

This should solve the merge problem where msftidy-failing modules still sneaks into the tree.
## Verification steps
- [x] Check out this feature branch.
- [x] `ln -sf ../../tools/dev/pre-commit-hook.rb ./.git/hooks/post-merge`
- [x] Create a temp branch: `git checkout -b temp-8779`
- [x] `git remote add -f todb-r7 git://github.com/todb-r7/metasploit-framework.git`
- [x] `git merge todb-r7/bug/8779-intentionally-bad-module`
- [x] Amaze at the error message.
- [x] Checkout this feature branch again (iow get out of `temp-8779`)
- [x] Land this pull request.

After this lands, Travis should also be able to deal with merges that contain msftidy-failing modules. See here:

https://travis-ci.org/todb-r7/metasploit-framework/builds/22202066

More could be done with the post-merge hook, of course. Namely, we'll want to (pretty soon) also warn about crappy merge messages that aren't following [50/72 formatting](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), or even start complaining about unsigned merges. Won't that be the day!
